### PR TITLE
Enabling hostname overrides

### PR DESCRIPTION
**Pull request by MovieStoreGuy**

### CODE REVIEW

File: cmd/tester/server.go [hunk 1]
```diff
@@ -106,9 +108,8 @@ func (s *Server) Run() error {
 	http.HandleFunc("/start", s.startHandler)
 	http.HandleFunc("/stop", s.stopHandler)
 	http.HandleFunc("/exit", s.exitHandler)
-	http.ListenAndServe(s.WebAddr, nil)
 
-	return nil
+	return http.ListenAndServe(s.WebAddr, nil)
```
Comment (MovieStoreGuy) [inline cmd/tester/server.go]: We were ignoring the error returned from the `ListenAndServe`, is this a worthwhile change?

Comment (tiedotguy) [reply cmd/tester/server.go]: Try to keep unrelated changes to a minimum. The tester is also a tester, and doesn't need the same level of care.

Comment (MovieStoreGuy) [reply cmd/tester/server.go]: Cool, I have reverted it back.

File: pkg/statsd/statsd_defaults_and_params.go [hunk 2]
```diff
@@ -32,6 +32,9 @@ var DefaultTags = gostatsd.Tags{}
 // DefaultInternalTags is the default list of additional tags on internal metrics
 var DefaultInternalTags = gostatsd.Tags{}
 
+// DefaultHostname will be set to the default hostname of the system
+var DefaultHostname = getHost()
```
Comment (tiedotguy) [inline pkg/statsd/statsd_defaults_and_params.go]: This becomes a global thing, which we don't like.  It should be called from `AddFlags`

**Pull request merged**